python/whylogs/api/whylabs/auth_file.py

```diff
@@ -0,0 +1,39 @@
+import configparser
+from pathlib import Path
+
+
+def _create_blank_auth_file(auth_path: Path) -> Path:
+    try:
+        # creates directory and file
+        auth_path.parent.mkdir(parents=True, exist_ok=True)
+        auth_path.touch(exist_ok=True)
+
+        # creates empty config file
+        config = configparser.ConfigParser()
+        config.read(auth_path)
+        config["whylabs"] = {}
+
+        # writes config file to disk
+        with open(auth_path, "w") as configfile:
+            config.write(configfile)
+        return auth_path
+
+    except (configparser.Error, OSError, IOError, PermissionError) as e:
+        raise ValueError(f"Error creating auth file at {auth_path}: {str(e)}") from e
```

Comment:
Shouldn't you warn instead of raise an error here? Otherwise the user can't fall back to the env variables